playbooks/roles/ocp-logging-tests-private-e2e/tasks/main.yml

```diff
@@ -0,0 +1,133 @@
+---
+
+# tasks file for ocp4 e2e
+- name: Install required packages
+  package:
+    name:
+      - git
+    state: latest
+
+- name: install the 'Development tools' package group
+  package: name="@Development tools" state=latest
+  when:  ansible_distribution == 'CentOS' or
+         ansible_distribution == 'RedHat' or
+         ansible_distribution == 'Fedora'
+
+- name: install the 'Build Essential' package group
+  package: name="build-essential" state=latest
+  when: ansible_distribution == 'Debian' or
+        ansible_distribution == 'Ubuntu'
+
+- name: Set test and results directory path
+  set_fact:
+    test_dir: "{{ ocp_logging_tests_private_e2e_dir }}src/github.com/openshift-tests-private"
+    results_dir: "{{ ansible_env.HOME }}/ocp_logging_tests_private_e2e_results"
+    bin_path: "{{ ocp_logging_tests_private_e2e_dir }}src/github.com/openshift-tests-private"
+    repository_command: ""
+
+- name: Create test directory
+  file:
+    path: "{{ test_dir }}"
+    state: "{{ item }}"
+    mode: '0755'
+  with_items:
+  - absent
+  - directory
+
+- name: Include role for installation of Go lang
+  include_role:
+    name: golang-installation
+  vars:
+    go_tarball: "{{ golang_tarball_logging_e2e }}"
+    golang_path: "/usr/local"
+
+- name: Clone e2e tests repo
+  git:
+    repo: "https://{{ githubuser | urlencode }}:{{ githubpassword | urlencode }}@github.com/openshift/openshift-tests-private.git"
+    dest: "{{ test_dir }}"
+    version: "{{ ocp_logging_tests_private_e2e_git_branch }}"
+
+- name: Run make build command at target
+  shell: make
+  environment: "{{ logging_e2e_env }}"
+  args:
+    chdir: "{{ test_dir }}"
+
+- name: Check if the binary is created
+  shell: ls -hl {{ test_dir }}/bin/extended-platform-tests | wc -l
+  args:
+    chdir: "{{ test_dir }}"
+  register: bin_output
+  failed_when: bin_output.stdout|int != 1
+
+- name: Copy openshift-tests-private binary
+  copy:
+    src: "{{ bin_path }}"
+    remote_src: true
+    dest: "/usr/local/bin/"
+    mode: '0755'
+
+- name: Create results directory
+  file:
+    path: "{{ results_dir }}"
+    state: "{{ item }}"
+    mode: '0755'
+  with_items:
+  - absent
+  - directory
+
+# Check Nodes status
+- name: Check all nodes are Ready
```

Comment:
Can we use this role [check-cluster-health](https://github.com/ocp-power-automation/ocp4-playbooks-extras/blob/main/playbooks/roles/check-cluster-health) ? It checks the cluster health.

Reply:
Yes, used the role check-cluster-health